xbmc/music/MusicDatabase.cpp

```diff
@@ -3794,11 +3794,17 @@ bool CMusicDatabase::GetSongsFullByWhere(const std::string &baseDir, const Filte
     unsigned int time = XbmcThreads::SystemClockMillis();
     int total = -1;
 
-    std::string strSQL = "SELECT %s FROM songview ";
-    if (artistData) 
+    std::string strSQL = "SELECT songview.* FROM songview ";
+    if (artistData)
```

Comment:
True. I started trying to change as little as possible so it was obvious what was happening, but that does lead to repetition.